api/api.go

```diff
@@ -292,6 +295,27 @@ func proxyContainer(c *context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// Proxy a request to the right node
+func proxyImage(c *context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Not sure how this would work - we only proxy to the first node containing the image.

I assume inspect would be slightly different depending on the node?

For instance, two nodes might have different images therefore the inspect would be different.